### PR TITLE
Hotfix settlem csv

### PR DESCRIPTION
**Pull request by 00eadeyemi**
### Features 

- Changes for settlement report download button to directly to call csv api

Requires https://github.com/PayBreak/merchant-api/pull/75
Requires https://github.com/PayBreak/paybreak-sdk-php/pull/27
Requires `Paybreak/paybreak-sdk-php` update

[Delivers #134525555](https://www.pivotaltracker.com/story/show/134525555)

### CODE REVIEW

**Comment by 00eadeyemi** [comment]
@wnowicki

**Comment by 00eadeyemi** [comment]
@EvanBarbour3

File: app/Providers/AppServiceProvider.php [hunk 1]
```diff
@@ -30,6 +30,10 @@ public function register()
             'App\Basket\Notifications\EmailLocationNotificationService'
         );
 
+        $this->app->when('PayBreak\Sdk\Gateways\SettlementCsvGateway')
+            ->needs('PayBreak\Sdk\ApiClient\ApiClientFactoryInterface')
+            ->give('App\Gateways\ApiCsvClientFactory');
+
```
Comment (EvanBarbour3) [inline app/Providers/AppServiceProvider.php]: Does this not need to be below the other ApiClientFactoryInterface?

Comment (00eadeyemi) [inline app/Providers/AppServiceProvider.php]: Doesn't matter where the new binding is placed but for better readability I will move it to the bottom Abstract -> Concrete

**Comment by 00eadeyemi** [comment]
@EvanBarbour3 re-review

**Pull request merged**